pkgs/tools/package-management/nix-update/default.nix

```diff
@@ -0,0 +1,38 @@
+{ lib
+, buildPythonApplication
+, fetchFromGitHub
+, glibcLocales
+, mypy
+, nix
+, nix-prefetch
+}:
+
+buildPythonApplication rec {
+  pname = "nix-update";
+  version = "0.1";
+
+  src = fetchFromGitHub {
+    owner = "Mic92";
+    repo = pname;
+    rev = version;
+    sha256 = "0mw31n7kqfr7fskkxp58b0wprxj1pj6n1zs6ymvvl548gs5rgn2s";
+  };
+
+  makeWrapperArgs = [
+    "--prefix" "PATH" ":" (lib.makeBinPath [ nix nix-prefetch ])
+  ];
+
+  checkInputs = [ glibcLocales mypy ];
+
+  checkPhase = ''
+    mypy nix_update
```

Comment:
I would drop this. Mypy upgrades sometimes break builds because tests gets more precise. I already run `mypy` in my repository, which means that all releases have been type-checked at some point. If you want a check wether everything works, just run `nix-update --help` as a smoke test.

Reply:
Changed to `nix-update --help`.